### PR TITLE
KK-446 | Let guardian edit their email address

### PR DESCRIPTION
**Pull request by badrange**

### CODE REVIEW

File: src/domain/home/form/partial/birthdateFormField.module.scss [hunk 1]
```diff
@@ -23,6 +23,13 @@
     display: grid;
     grid-template-columns: 1fr 0.2fr 1fr 0.2fr 1fr;
     align-items: baseline;
+
+    /* Hide number input spinners */
```
Comment (badrange) [inline src/domain/home/form/partial/birthdateFormField.module.scss:27]: This is unrelated, and was added by accident. It is desirable though.

Comment (Jokinen) [reply src/domain/home/form/partial/birthdateFormField.module.scss:27]: I think you are still able to see the spinners in FF. Noticed by accident.
